orchestration/dagster_orchestration/hca_orchestration/repositories.py

```diff
@@ -1,9 +1,11 @@
-from dagster import repository
+from typing import Union
```

Comment:
looking forward to the new python 3.10 syntax for union types (`Union[A, B]` -> `A | B`)